TACC/protx-dashboard
# Task/cooks 361 & 344 change analytics tab wording, clean up analytics components

## Overview: ##
Change wording on county analytics chart from "Positve" or "Negative" to "Positive correlation" or "Negative correlation"

## Related Jira tickets: ##

* [COOKS-361](https://jira.tacc.utexas.edu/browse/COOKS-361)
* [COOKS-344](https://jira.tacc.utexas.edu/browse/COOKS-344)

## Summary of Changes: ##

- Added "correlation" to negative or positive column on new analytics chart

- Cleaned up and removed old, unused Predictive Feature Table and Predictive Feature Data components. 

## Testing Steps: ##
1. Go to [cep.test/protx](https://cep.test/protx/dash/analytics)](https://cep.test/protx/dash/analytics) and click on a county
2. Check that the table says "Negative correlation" or "Positive correlation"

## UI Photos:
<img width="1525" alt="Screen Shot 2022-11-22 at 10 26 51 AM" src="https://user-images.githubusercontent.com/96220951/203368085-12dfb3a8-9de2-4af8-9115-667c0f99fcad.png">

## Notes: ##
